toolkit-distribution/src/release/licenses/LICENSE-jsch

```diff
@@ -0,0 +1,30 @@
+JSch 0.0.* was released under the GNU LGPL license.  Later, we have switched 
```

Comment:
@WillemJiang I'm not sure this is ok

Reply:
It's fine once the License is BSD, I just checked the we are using JSch 0.1.x which is BSD license.